2021q3/freebsd-foundation.adoc

```diff
@@ -88,6 +88,12 @@ entries:
 - Using OCF in WireGuard
 - syzkaller on FreeBSD
 
+Other Foundation-sponsored work:
+
+- Support for booting FreeBSD on the Arm Armv8-A AEM simulator
+- sha256 support to libmd(4)
```

Comment:
Should mention the sha256 improvement is on arm64 (or change the heading to mention as such)